main.rb

```diff
@@ -1,8 +1,15 @@
 require_relative 'user_interface'
+require_relative 'local_storage'
 
 def main
+  LocalStorage.book_file = 'local_storage/books.json'
+  LocalStorage.person_file = 'local_storage/people.json'
+  LocalStorage.rental_file = 'local_storage/rentals.json'
```

Comment:
- [ ] As required, the dummy data you added to the database was not loaded at the start of the app, even when I added some data and I exited, it still did not load. I would implore you to look into it as it is required.
![image](https://user-images.githubusercontent.com/86200356/232473830-2b195d44-84fa-4b74-bb0c-e52734953ee4.png)

And also add the case where there happens not to be data in the database. It can look something like this:
```
 def load_people
    # *the missing file case is being handled with this line
    return [] unless File.exists?("people.json")

    JSON.parse(File.read("people.json")).map do |person|
      if person["json_class"] == "Student"
        Student.new(person["age"], person["name"], person["parent_permission"])
      elsif person["json_class"] == "Teacher"
        Teacher.new(person["age"], person["specialization"], person["name"], person["parent_permission"])
      end
    end
  end

```

This is because I notice when your database is empty of data, your app does not run.

In the attached image, I removed the data you added to the database and I could not run the app.
![image](https://user-images.githubusercontent.com/86200356/232474831-afa24b0a-3d33-4f9e-9512-b64e367a5a4b.png)

Reply:
Hi @Olamarx, thanks for your review. We will look into this issue.